keps/prod-readiness/sig-auth/3926.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3926
```

Comment:
didn't we have KEP numbers >4000 already?

Reply:
By quite a bit.. since KEPs source numbers from the kubernetes/enhancements  PR/issues number pool now :)

Reply:
> didn't we have KEP numbers >4000 already?

The effort started prior to the current release.